src/main/java/org/folio/circulation/infrastructure/storage/sessions/PatronActionSessionRepository.java

```diff
@@ -151,6 +156,7 @@ private JsonObject mapToJson(PatronSessionRecord patronSessionRecord) {
     return findWithMultipleCqlIndexValues(patronActionSessionsStorageClient,
       PATRON_ACTION_SESSIONS, PatronSessionRecord::from)
       .findByIdIndexAndQuery(patronIds, PATRON_ID, actionTypeQuery)
+      .thenCompose(r -> r.after(this::fetchLoans))
```

Comment:
Do the endpoints that execute this code have the appropriate permissions to fetch loans?

Reply:
Yes, the endpoint `/circulation/notice-session-expiration-by-timeout` already has permission for `circulation-storage.loans.collection.get`

Reply:
Does `fetchLoans` only fetch the loan, or does it also fetch related records like items etc?

Reply:
It does fetch loan with item and loanPolicy